innersource-guidance-group.md

```diff
@@ -0,0 +1,43 @@
+## Title
+InnerSource Guidance Group
+
+## Patlet
+A company may require a mechanism to set broad governance and behavioral guidelines as part of the rollout and sustaining needs of an InnerSource initiative.
+
+## Problem
+Without a set of organizational standards Trusted Committers, and individual teams, are free to set their own standards, which is a step in the right direction.  However, team standards could be highly variant and further diverge over time as they evolve and adapt to new needs and forces.  This can lead to confusion and frustration as contributors submit code across team and functional boundaries. The business is interested establishing a set of agreed-upon standard operating procedures that guide teams while still providing sufficient autonomy to meet the parochial needs of individual teams.
+
+## Context
+A company is planning to introduce InnerSource as a strategic initiative. They are faced with reforming and steering the behavior of developers and engineers toward a common goal.  In general functional groups and teams within the company adhere to a broad spectrum of standards regarding tools, development languages, collaboration models, engagement models, documentation models, etc. The company recognizes the value of the Trusted Committer role as a situational leader for a team that sets and enforces team standards, code contribution standards, contribution agreements, PR management, etc. however the company is interested in establishing organizational guidelines as well that steer and focus Trusted Committers and the teams they represent.
```

Comment:
> A company is planning to introduce InnerSource

Is this point an essential point of context for using this pattern?  Seems like the pattern would apply even to those that have already introduced inner source, for example.